README.md

```diff
@@ -94,7 +94,10 @@
 #### System Monitor
    * Page Settings
       * `.local/share/plasma-systemmonitor/`: One page file per page
-	
+
+#### Krusader
```

Comment:
Is this a new entry in KDE Settings window? My idea was to follow its layout directly. Since I have not used KDE for a long time I cant check.

if it is a separate program, I think its better to add this near end of the file. just above https://github.com/shalva97/kde-configuration-files#changing-appearance add 
```
# KDE Applications

#### Krusader
   * `.config/krusaderrc`
```

Reply:
Its not in the settings, its a separate app (although very old, kde3 or older). Ive applied the suggestion, nice cheatsheet btw, thx.